src/chrome/content/rules/Wdc.com.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="Wdc.com">
+  <target host="wdc.com" />
+  <target host="www.wdc.com" />
+
+  <rule from="^http://(www\.)?wdc\.com/" to="https://www.wdc.com/" />
```

Comment:
This server could not prove that it is www.wdc.com; its security certificate is from a248.e.akamai.net. This may be caused by a misconfiguration or an attacker intercepting your connection.